src/app.ts

```diff
@@ -196,6 +196,7 @@ export async function handleKnownVoterBlockLogic(
           parentMessageTs: userInfo[userInfo.activeChannelId],
           channel: userInfo.activeChannelId,
           isVoterMessage: true,
+          isBlocked: true,
         },
         inboundDbMessageEntry,
         userInfo
```

Comment:
I'd like to keep SlackApiUtil.sendMessage as simple as possible. Can we move the updateThreadStatusFromMessage logic here (and elsewhere if needed) instead of passing an isBlocked?